wrangler-core/src/main/java/co/cask/directives/date/DiffDate.java

```diff
@@ -46,7 +46,8 @@
   private String column1;
   private String column2;
   private String destCol;
-  private final Date date = new Date();
+  // Timestamp now in system default timezone
+  private final ZonedDateTime date = ZonedDateTime.now();
```

Comment:
it doesn't look like this is used

Reply:
Its used here: https://github.com/data-integrations/wrangler/blob/develop/wrangler-core/src/main/java/co/cask/directives/date/DiffDate.java#L89-L91